RELEASE.md

```diff
@@ -73,24 +73,18 @@ Highlights:
     $ git checkout -b <branch>
     ```
 
-1. Remove the `-dev` suffix from the version number found in the `VERSION` file. *Note*: there must not be a space after the `-i`.
-
-    ```
-    $ sed -i'' -e 's/-dev//' VERSION
```

Comment:
When I ran this on my Mac, I ended up with a file called `VERSION-e` containing `0.62.0-dev`. I think it's clear enough that we say "remove the -dev" and leave it at that. 😄

Reply:
I think the `sed` that comes with macOS responds to different options than the GNU one on Linux.

That's fair; pretty soon Expeditor will be doing this for us, anyway. 😄